cmd/gateway-main.go

```diff
@@ -36,36 +36,30 @@ USAGE:
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}{{end}}
+BACKEND:
+  azure: Microsoft Azure Blob Storage. Default ENDPOINT is https://core.windows.net
```

Comment:
The URL **https://core.windows.net/** is not accessible on browser. Is that expected?